src/kernels/jupyter/launcher/notebookProvider.ts

```diff
@@ -88,29 +85,13 @@ export class NotebookProvider implements INotebookProvider {
             await this.jupyterNotebookProvider.connect(serverOptions);
         }
         Cancellation.throwIfCanceled(options.token);
-        trackKernelResourceInformation(options.resource, {
-            kernelConnection: options.kernelConnection,
```

Comment:
Moved this into the Kernel.ts class as thats the top most layer where we start the kernels
Then its easier to look for these